README.md

```diff
@@ -199,6 +199,60 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+To use the xray calls directly with lambda - for use with sql for example - you must begin a new facade segment and use the resulting context in relevant context-enabled calls.
+
+```go
+package yourpkg
+
+import (
+    "context"
+    "fmt"
+    "github.com/aws/aws-xray-sdk-go/header"
+    "github.com/aws/aws-xray-sdk-go/xray"
+    "time"
+)
+
+func HandleRequest(ctx context.Context, id int) (string, error) {
+
+    facadeSegmentName := "facade"
+    traceIDstr, isTraced := ctx.Value("x-amzn-trace-id").(string)
+
+    msg := "Request not traced"
+
+    if isTraced {
+        msg = fmt.Sprintf("request traced with %s", traceIDstr)
+
+        header := header.FromString(traceIDstr)
+        newCtx, seg := xray.BeginFacadeSegment(ctx, facadeSegmentName, header)
+        if seg == nil {
+            return "error", fmt.Errorf("couldn't make segment")
+        }
+        defer seg.Close(nil)
+
+        db, err := xray.SQL("postgres", "postgres://user:password@host:port/db")
+        if err != nil {
+            return "error", err
+        }
+        defer db.Close()
+
+        var username string
+        var created time.Time
+        err = db.QueryRow(newCtx, "SELECT username, created_at FROM users WHERE id=?", id).Scan(&username, &created)
```

Comment:
at here, you don't need to inject `newCtx`. Just use the `ctx` from `HandleRequest(ctx context.Context, id int)` and it will start generate subsegment.